README.md

```diff
@@ -1 +1,5 @@
-# did-sdk-js
\ No newline at end of file
+# did-sdk-js
+Support for the Hedera Hashgraph DID Method and Verifiable Credentials on the Hedera JavaScript/TypeScript SDK
+
+# Tests
+For run tests you need to create and fill ```test/variables.js``` file before.
```

Comment:
I'd like to see this README fleshed out a bit more. I attempted to run the tests, but the test/variables.js file is missing. I remembered seeing it in the README, but there isn't enough information in here for me to be able to know what to fill the file with. Please update the README with comprehensive documentation so I can build and test.

Reply:
I just saw the variables.js.sample. It would be good to document the existence of the file and so forth